ui/widgets/nuxeo-date-picker.js

```diff
@@ -253,6 +267,12 @@ import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
       }
       this._preventInputUpdate = false;
     }
+
+    _firstDayOfWeekChanged() {
+      if (this.firstDayOfWeek) {
```

Comment:
This won't work if `this.firstDayOfWeek` equals 0:
```suggestion
      if (this.firstDayOfWeek != null) {
```

Reply:
Thank you! 👍